dali/pipeline/operators/op_schema.h

```diff
@@ -300,6 +309,10 @@ class DLL_PUBLIC OpSchema {
     return num_output_;
   }
 
+  DLL_PUBLIC inline bool AllowsSyncSeeds() const {
+    return allow_sync_seeds_;
+  }
+
   DLL_PUBLIC inline bool AllowsMultipleInputSets() const {
```

Comment:
What's the difference between those two? Do we _need_ to have AllowSyncSeeds at all?

Reply:
It's a little bit different, as we won't be explicitly handling the multiple input sets on C++ level.
After this PR I will rebase #1088  and remove the *MultipleInputSet() related family of functions.